parent/pom.xml

```diff
@@ -306,6 +306,25 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>net.revelc.code.formatter</groupId>
+                    <artifactId>formatter-maven-plugin</artifactId>
+                    <version>2.9.0</version>
+                    <executions>
+                        <execution>
+                            <id>verify-code-formatting</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>validate</goal>
+                            </goals>
+                            <configuration>
+                                <configFile>${formatter.config}</configFile>
```

Comment:
I think we should rather refer to the formatting file here with `${project.build.directory}/eclipse-formatter.xml`, and remove the config from the different POMs so it's only defined once in the parent POM. I'd also move this to the `properties` section (like for the `jacoco.data.file` property).